cmd/exec/main.go

```diff
@@ -591,11 +600,13 @@ Will update the helloworld workflow and set the remote workflow variable 'data.j
 }
 
 var eventCmd = &cobra.Command{
-	Use:   "event Event_PATH",
+	Use:   "event EVENT_PATH",
 	Short: "Remotely trigger direktiv event with local files",
 	Long: `Remotely trigger direktiv event with local files. This process will trigger and create the given Event when its not already existing.
 
-EXAMPLE: direktiv-sync event greeting.yaml --addr http://192.168.1.1 --namespace admin --path greeting
+EXAMPLE: direktiv-sync event greeting.yaml 
+
+event command have also optional flags like --source, --type, --specversion, --id to create or overwrite keys in given JSON file
```

Comment:
Listing the flags here is okay, but not necessary. The auto-generated `--help` info will include them.